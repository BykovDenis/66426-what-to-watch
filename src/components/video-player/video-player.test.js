import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import VideoPlayer from './index.jsx';

describe(`VideoPlayer correctly renders after relaunch`, () => {
  const props = {
    preview: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    img: {
      fileName: `fantastic-beasts-the-crimes-of-grindelwald`,
      extension: `jpg`,
    },
  };

  const tree = renderer
    .create(
      <MemoryRouter initialEntries={['/']}>
        <VideoPlayer {...props} />
      </MemoryRouter>
    )
    .toJSON();

  it(`Component should render`, () => {
    expect(tree).toMatchSnapshot();
  });
});
