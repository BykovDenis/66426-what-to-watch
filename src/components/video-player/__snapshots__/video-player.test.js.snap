// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer correctly renders after relaunch Component should render 1`] = `
<video
  autoPlay={true}
  height={150}
  poster="../img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
  width={300}
>
  <source
    src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
  />
</video>
`;
